cocaine/proxy/helpers.py

```diff
@@ -50,16 +54,43 @@ def __init__(self, endpoints):
                 raise ValueError("Endpoint has to begin either unix:// or tcp:// %s" % i)
 
 
+def write_chunk(request, chunk):
+    request.connection.write(SIZE_OF_CHUNK_FMT.format(len(chunk)))
+
+    chunk += CRLF
+    request.connection.write(chunk)
```

Comment:
It seems that connection is of HTTP1Connection type, and don't provide flush method ((, but seems (by _strace_ output) that it do some buffering internally and flushes it time to time on some reasonable timer based events.